en/translation.json

```diff
@@ -620,5 +623,7 @@
   "eth_gas_price": "ETH gas price",
   "strk_gas_price":"STRK gas price",
   "eth_gas_price_for_block": "ETH gas price for block",
-  "strk_gas_price_for_block": "STRK gas price for block"
+  "strk_gas_price_for_block": "STRK gas price for block",
+  "l1->l2" : "L1->L2",
```

Comment:
No special symbols in key, since we want to keep everything snake case only `_` are allowed.